specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -1626,6 +1626,11 @@
           },
           "default": "Microsoft.Storage"
         },
+        "requireInfrastructureEncryption": {
+          "type": "boolean",
```

Comment:
As you shown in examples, it looks there is a default value `false` in server side for the property, right?

Reply:
This defaults to null. Corrected the example.